### DISCUSSION TITLE
Dashboard Menu Order not affecting Pods Admin

### DISCUSSION THREAD

**Discussion by OWMC** [pods-framework/pods#6638] [Q&A]
This is only affecting new builds (for all my previous WP sites this works fine).

I always reorder my admin menu so that Pods Admin sits toward the top, next to Settings. However, having tried multiple fresh installs with fresh databases and no other plugins, this code no longer seems to work. I've tried it as a MU-Plugin and separately just in my theme's functions.php (I've been using a fresh copy of Underscores theme and a fresh copy of HTML5Boilerplate).

The code does work insofar as it affects all the other menu items, it's only Pods Admin which doesn't take affect, staying stuck at the bottom of the menu.

`
function custom_menu_order($menu_ord) {
  if (!$menu_ord) return true;
      return array(
      'index.php', 
      'tools.php', 
      'options-general.php',
      'starter_pack',
      'pods',
      'themes.php', 
      'plugins.php' 
  );
}
add_filter('custom_menu_order', 'custom_menu_order'); 
add_filter('menu_order', 'custom_menu_order');  `

**Comment by sc0ttkclark** [comment]
We add our menus on the admin_menu action at priority 99. Beyond that, we don't do any special menu ordering.

I'm not sure what changed, but perhaps you should try using a higher priority on your filter?

**Comment by OWMC** [comment]
Tried altering my filter priority as well as modifying the filter priority for admin_menu in pods. No joy.

(My gut tells me this isn't a bug with Pods, but it's so strange! Going to try cloning older sites where the function works later. If I resolve this I'll post back.)
